components/automate-ui/src/app/modules/infra-proxy/policy-groups/policy-groups.component.ts

```diff
@@ -48,9 +48,9 @@ export class PolicyGroupsComponent implements OnInit, OnDestroy {
       this.store.select(getGroupsStatus),
       this.store.select(policyFile)
     ]).pipe(takeUntil(this.isDestroyed))
-    .subscribe(([ getPolicyFilesSt, allPolicyFilesState]) => {
-      if (getPolicyFilesSt === EntityStatus.loadingSuccess && !isNil(allPolicyFilesState)) {
-        this.policyGroups = allPolicyFilesState.policyFile;
+    .subscribe(([ getPolicyFilesSt, policyFilesState]) => {
+      if (getPolicyFilesSt === EntityStatus.loadingSuccess && !isNil(policyFilesState)) {
+        this.policyGroups = policyFilesState;
```

Comment:
i mean to change the policy group to policy files and in the you are sending it it will have as policyfile name not group

Reply:
I have updated the code.